charts/karpenter/templates/post-install-hook.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ .Release.Name }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "karpenter.labels" . | nindent 4 }}
+  {{- with .Values.additionalAnnotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+    "helm.sh/hook": post-install
+    "helm.sh/hook-delete-policy": hook-succeeded
+    "helm.sh/hook-delete-policy": hook-failed
+spec:
+  ttlSecondsAfterFinished: 10
```

Comment:
Is there a reason we care about if it's up after it finished? If we just leave it alone, the user can clean it up when they want to, or it'll just get garbage collected eventually.

Reply:
From my testing it will not get garbage collected, unless we set the TTL. I can a use case for debug, but if succeed   I think we should remove the pod from the cluster

Reply:
Seems like we can just set it to 0 then